src/styles/blocks/day.scss

```
@@ -0,0 +1,23 @@
+@import "../utils/function.scss";
+
+.calendar-day {
+  width: 100px;
+  height: 100px;
+  background-color: #eee;
+  border: 1px solid black;
+  transition: color 0.5s, transform 0.5s;
```

Don't forget about the variables
```suggestion
  border: 1px solid $c-black;
transition: color $transitionDuration, transform $transitionDuration;
```